source/install/install-ubuntu-1604.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-ubuntu-1604:
+
+=========================================
+Installing Mattermost on Ubuntu 16.04 LTS
+=========================================
+
+Install a production-ready Mattermost system on 1 to 3 machines.
+
+.. include:: install-common-intro.rst
+
+.. contents:: Install and configure the components in the following order:
```

Comment:
I think this is missing the `Note that you need only one database, either MySQL or PostgreSQL.` note that other install docs have.